README.md

```diff
@@ -1637,122 +1634,147 @@ func ==(lhs: Item, rhs: Item) -> Bool {
     return lhs.GUID == rhs.GUID
 }
 
+enum Result<T> {
```

Comment:
I had figured out you had something like this but could not find it. I will fix this